server/src/main/kotlin/suwayomi/tachidesk/manga/MangaAPI.kt

```diff
@@ -132,5 +133,14 @@ object MangaAPI {
             get("summary", UpdateController.updateSummary)
             ws("", UpdateController::categoryUpdateWS)
         }
+
+        path("track") {
+            get("list", TrackController.list)
+            post("login", TrackController.login)
+            post("logout", TrackController.logout)
+            post("search", TrackController.search)
+            post("bind", TrackController.bind)
+            post("update", TrackController.update)
+        }
```

Comment:
Do you think you would be able to add GraphQL endpoints for these?

Reply:
Hi @Syer10, I'm sorry that I have limited knowledge about GraphQL. I only know that it is a way to query data. Among the six apis, the first one is used for querying data, while the others are for submitting data, which may not be suitable.

Reply:
I'll try to take a look at implementing it when I have time, I'll push to this branch so that it can be included in the PR.
We have deprecated the REST api and all future development is to take place in the GraphQL api(under the suwayomi.tachidesk.graphql package). So we need GraphQL endpoints for these before merging.

Reply:
BTW, the related PR to Sorayomi can be found here: https://github.com/Suwayomi/Tachidesk-Sorayomi/pull/236